### ISSUE TITLE
Deactivate method and file name obfuscation

### ISSUE THREAD

**Issue by Turysaz** [patzly/grocy-android#468]
Currently, symbols are obfuscated by ProGuard during the release build.
Because of that, stack traces in the logs are hardly helpful, like in #451 or #425.

I think, since this is a FOSS app anyways, the obfuscation should be disabled to make the fixing of bugs easier.

**Comment by MrJohnZoidberg** [comment]
Mmh this is a good idea for development, we've originally enabled Proguard to minimize the app size and the default proguard arguments also enables obfuscation...
I don't know whether obfuscation helps decreasing app size, but I also think it makes developement hard sometimes. So I will disable it (I will try [this post](https://stackoverflow.com/a/55785351)).

**Comment by MrJohnZoidberg** [comment]
Closed with f8726f8

**Comment by Turysaz** [comment]
Wow, that was fast, thanks!

I neither know ProGuard nor Gradle very well and hence cannot tell which way is the right way&trade; to achieve this, but I found the following lines in the `proguard-rules.pro` file:

```
# Uncomment this to preserve the line number information for
# debugging stack traces.
#-keepattributes SourceFile,LineNumberTable

# If you keep the line number information, uncomment this to
# hide the original source file name.
#-renamesourcefileattribute SourceFile
```

So in case f8726f89600414513658b5727d1edd787340dcf9 does not do the job properly, that could be worth keeping in mind.
